src/Parser/DisciplinePriest/CombatLogParser.js

```diff
@@ -170,6 +182,17 @@ class CombatLogParser extends MainCombatLogParser {
           </dfn>
         )}
       />,
+      this.modules.atonement.active && (
+        <StatisticBox 
+        icon={<SpellIcon id={SPELLS.ATONEMENT1.id} />}
+        value= {this.modules.atonement.improperAtonementRefreshes.length}
+        label={(
+          <dfn data-tip={`${((this.modules.atonement.improperAtonementRefreshes.length / this.modules.atonement.totalAtonementRefreshes * 100) || 0).toFixed(2)}% (${this.modules.atonement.improperAtonementRefreshes.length}/${this.modules.atonement.totalAtonementRefreshes}) of your Atonement refreshes were too early. ${((this.modules.atonement.improperAtonementRefreshes.length / this.modules.atonement.totalAtones * 100) || 0).toFixed(2)}% (${this.modules.atonement.improperAtonementRefreshes.length}/${this.modules.atonement.totalAtones}) of all Atonements applied were refreshed too early.` }>
```

Comment:
Is the percentage of early vs total refreshes something that's important? My suggestion for the tooltip would be:

The amount of Atonements that were refreshed earlier than within 3 seconds of the buff expiring. You applied Atonement `200` times in total, `30` (`15`%) of them were refreshes of existing Atonements, and `10` (`5`%) of them were considered early.

This explains what it means, and provides all the info in an easy to understand format.